MiniApp/Classes/MiniAppLister.swift

```diff
@@ -1,2 +1,54 @@
 internal class MiniAppLister {
+    var environment: Environment
+    var miniAppClient: MiniAppClient
+
+    init(environment: Environment) {
+        self.environment = environment
+        self.miniAppClient = MiniAppClient()
+    }
+
+    func fetchList(completionHandler: @escaping (Result<[MiniAppInfo], Error>) -> Void) {
+        guard let url = environment.listingUrl else {
```

Comment:
I'm not sure that `Environment` is the right place for `listingUrl`. The `baseUrl` is part of the environment, but the listing URL is part of the API request, so it should closer to the API part of the code.

Reply:
agree with Cory

Reply:
Done.